src/wp-admin/includes/class-wp-privacy-requests-table.php

```diff
@@ -527,7 +527,7 @@ public function column_email( $item ) {
 	}
 
 	/**
-	 * Next steps column. Overridden by children.
+	 * Returns the markup for the Next steps column. Overridden by children.
```

Comment:
Remove capital N on "Next"

Reply:
Fixed in [e3f2e97](https://github.com/WordPress/wordpress-develop/pull/4552/commits/e3f2e9783fd2a232f4275a6abeea815c1b133a07).